src/VisualStudio/Xaml/Impl/CodeFixes/AddMissingNamespace/XamlAddMissingNamespacePrefixCodeFixProvider.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editor.Xaml.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.Xaml.Features;
+
+namespace Microsoft.CodeAnalysis.Editor.Xaml.CodeFixes.AddMissingNamespace
+{
+    [ExportCodeFixProvider(StringConstants.XamlLanguageName, Name = PredefinedCodeFixProviderNames.AddUsingOrImport), Shared]
+    [ExtensionOrder(After = PredefinedCodeFixProviderNames.AddMissingReference)]
+    internal class XamlAddMissingNamespacePrefixCodeFixProvider : CodeFixProvider
```

Comment:
Yes, that was my thought here. For every other case but this one, you're connecting to a not-really-stable API that we don't know how good it is, and so it's entirely reasonable we have shims to decouple things. But this API is shipped, so why not just put the code in your real assembly and be done with it?